### ISSUE TITLE
Update Flutter Logo on docs.flutter.dev to SVG

### ISSUE THREAD

**Issue by bernaferrari** [flutter/website#4294]
It seems Flutter doesn't have a real SVG logo.

The website is using an SVG file that has a PNG image inside:
```<image overflow="visible" opacity="0.2" enable-background="new    " width="127" height="99" xlink:href="data:image/png;base64,....```

Related: https://github.com/flutter/website/issues/2601.

@maguinis tells about a [drive directory](https://drive.google.com/drive/folders/1NWJGmv_j9f0HEVWYi0NAEUPzu2l0Verp) with all the logos, but they are wrong too!

Upon Googling, I could find [this](https://www.vectorlogo.zone/logos/flutterio/index.html), but some positions and sizes are wrong (slightly, but it is). 

Tl;Dr: no one knows how Flutter's logo looks like in pure SVG.

**Comment by johnpryan** [comment]
That drive directory is the official source for the logos. You might take a look at the Adobe Illustrator files in this directory: https://drive.google.com/corp/drive/folders/1yaaGcIdNkba6VYT4xIwbvuwhHXzcQYov, but I'm not sure if AI will export them using PNG too.

**Comment by bernaferrari** [comment]
Upon further inspection, including the AI files, I found out the issue.

There is something which is exported as an image, in the "second slash of the F". I'm not exactly sure what it is, and comparing both I couldn't spot any difference.

I attached the original and the modified (without the image inside it). It is 6kb, vs 13kb! Half the size!

[flutter icons.zip](https://github.com/flutter/website/files/4881536/flutter.icons.zip)

![image](https://user-images.githubusercontent.com/351125/86659762-7ec9c580-bfc0-11ea-8479-878e5444f08c.png)
